.evergreen/config.yml

```diff
@@ -1055,6 +1090,17 @@ buildvariants:
       # macOS MongoDB servers do not staple OCSP responses and only support RSA.
       - name: ".ocsp-rsa !.ocsp-staple"
 
+- matrix_name: "versioned-api-tests"
+  matrix_spec:
+    os-fully-featured: ubuntu-18.04
```

Comment:
since the point is just to test versioned API, it seemed probably fine to me to limit this to a single platform and Swift version. what do you think?

Reply:
Seems fine to me